dev/run_gitlab_in_docker.sh

```diff
@@ -78,7 +78,7 @@ fi
 cecho b "Starting GitLab..."
 # run GitLab with root password pre-set and as many unnecessary features disabled to speed up the startup
 docker run --detach \
-    --hostname gitlab.foobar.com \
+    --hostname localhost \
```

Comment:
Added to avoid python-gitlab warnings, as it respects gitlab's `external_url` for pagination. See https://github.com/python-gitlab/python-gitlab/blob/1020ce965ff0cd3bfc283d4f0ad40e41e4d1bcee/gitlab/client.py#L589-L594

https://docs.gitlab.com/ee/api/#pagination-link-header